sdk/quantum/quantum-jobs/rollup.base.config.js

```diff
@@ -0,0 +1,170 @@
+    external: depNames.concat(externalNodeBuiltins),
+    output: {
+      file: "dist/index.js",
+      format: "cjs",
+      sourcemap: true,
+      banner: banner,
+      name: "azurequantumjobs"
+    },
+    preserveSymlinks: false,
+    plugins: [
+      sourcemaps(),
+      replace({
+        delimiters: ["", ""]
+      }),
+      nodeResolve({ preferBuiltins: true }),
+      cjs()
+    ]
+  };
+
+  if (test) {
+    // Entry points - test files under the `test` folder(common for both browser and node), node specific test files
+    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/**/*.spec.js"];
+    baseConfig.plugins.unshift(
+      multiEntry({ exports: false }),
+      json() // This allows us to import/require the package.json file, to get the version and test it against the user agent.
+    );
+
+    // different output file
+    baseConfig.output.file = "test-dist/index.node.js";
+
+    // mark assert packages we use as external
+    baseConfig.external.push("assert");
+
+    baseConfig.external.push(...Object.keys(pkg.dependencies), ...Object.keys(pkg.devDependencies));
+
+    // Disable tree-shaking of test code.  In rollup-plugin-node-resolve@5.0.0, rollup started respecting
+    // the "sideEffects" field in package.json.  Since our package.json sets "sideEffects=false", this also
+    // applies to test code, which causes all tests to be removed by tree-shaking.
+    baseConfig.treeshake = false;
+  } else if (production) {
+    baseConfig.plugins.push(terser());
+  }
+
+  return baseConfig;
+}
+
+export function browserConfig(test = false) {
+  const baseConfig = {
+    input: input,
+    output: {
+      file: "dist-browser/quantumjobs.js",
+      format: "umd",
+      name: "Azure.QuantumJobs",
+      globals: {
+        "@azure/core-http": "Azure.Core.HTTP"
+      },
+      sourcemap: true
+    },
+    external: ["fs-extra"],
+    preserveSymlinks: false,
+    plugins: [
+      sourcemaps(),
+      replace({
+        delimiters: ["", ""]
+      }),
+
+      nodeResolve({
+        mainFields: ["module", "browser"],
+        preferBuiltins: false
+      }),
+      cjs({
+        namedExports: {
+          chai: ["assert", "expect", "use"],
+          assert: ["ok", "equal", "strictEqual", "deepEqual", "fail", "throws", "notEqual"],
+          events: ["EventEmitter"],
+          "@opentelemetry/api": ["CanonicalCode", "SpanKind", "TraceFlags"]
+        }
+      }),
+
+      inject({
+        modules: {
+          process: "process"
+        },
+        exclude: ["./**/package.json"]
+      }),
+
+      json()
+    ]
+  };
+
+  baseConfig.onwarn = ignoreKnownWarnings;
+
+  if (test) {
+    baseConfig.input = ["dist-esm/test/*.spec.js", "dist-esm/test/internal/*.spec.js"];
+
+    baseConfig.external.unshift(...["process"]);
+
+    baseConfig.output.globals = {
+      ...baseConfig.output.globals,
+      fs: "fs-extra",
+      "fs-extra": "fs",
+      process: "process",
+      path: "path"
+    };
```

Comment:
```suggestion
```

I'm not sure if you need this, can you just verify the tests don't fail if this is deleted?

Reply:
For the simple test case that I have added now, I am pretty sure I do not need them.